tools/partition_graph.py

```diff
@@ -263,6 +266,25 @@
             raise Exception('There is no predicted edge type to split. Please set the '
                             +'target_etype argument ......')
 
+    if args.no_split:
+        if pred_ntypes is not None:
+            for ntype in pred_ntypes:
+                if 'train_mask' in g.nodes[ntype].data:
+                    del g.nodes[ntype].data['train_mask']
```

Comment:
why do you need to delete train_mask? does the train mask exist in the original graph data?

Reply:
The answer is yes for some datasets. So if no_split is set to True, we need to remove these masks.